docs/arch/adr-57.md

```diff
@@ -0,0 +1,18 @@
+# ADR 57: build @zooniverse/classifier and @zooniverse/react-components as ES modules
+
+## Status
+Accepted
+
+## Context
+The classifier and React components libraries were originally built with Webpack. Each was built as a single large UMD `main.js` file.
+
+Since the monorepo was firsted started, ES modules have become the standard for JS packages. Recent versions of Webpack can [tree-shake ESM](https://webpack.js.org/guides/tree-shaking/) to remove unused code. CommonJS is used for legacy package support, and increasingly deprecated for newer packages. All modern browsers, and NodeJS, support `import/export`.
```

Comment:
```suggestion
Since the monorepo was first started, ES modules have become the standard for JS packages. Recent versions of Webpack can [tree-shake ESM](https://webpack.js.org/guides/tree-shaking/) to remove unused code. CommonJS is used for legacy package support, and increasingly deprecated for newer packages. All modern browsers, and NodeJS, support `import/export`.
```